client/VCMI_client.cbp

```diff
@@ -13,6 +13,7 @@
 				<Option object_output="../obj/Client/Debug/x86" />
 				<Option type="1" />
 				<Option compiler="gcc" />
+				<Option parameters="--testmap=Maps/Reclamation --spectate --spectate-skip-battle --spectate-ignore-hero" />
```

Comment:
Is it an intended change?

Reply:
Yes it is.

Reply:
I decided it is better to have default run parameters for debug to be automated run for now.